cmd/eval.go

```diff
@@ -88,3 +36,52 @@ func validateFile(file []byte) error {
 	var holder interface{}
 	return yaml.Unmarshal(file, &holder)
 }
+
+var (
+	evalCmd = &cobra.Command{
+		Use:   "eval [file]",
+		Short: "Evaluate the combinations for a file at the given path",
+		Long: `
+Evaluate the combinations for a file at the given path. The file provided must be valid YAML.
+
+Note: the combo binary requires the --replacement flag to be explicitly set.
+
+The replacements flag allows users to specify a series of key value pairs in the form of KEY=VALUES.
+
+Example: combo eval -r REPLACE_ME=1,2,3 path/to/file
+	`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			replacements, _ := cmd.Flags().GetStringToString("replacements")
+
+			file, err := os.ReadFile(args[0])
```

Comment:
This will produce a panic if no input file has been specified. We can enforce this at the command-level through the `ExactArgs` parameter.